server/schema_syncer.go

```diff
@@ -17,6 +17,10 @@ const (
 	schemaSyncInterval = time.Duration(30) * time.Minute
 )
 
+var (
+	instanceSyncChan chan *api.Instance = make(chan *api.Instance, 100)
```

Comment:
I think we can run the syncer in another go routine so that the buffer size won't matter.
```go
// in the main loop
for {
    select {
        case <-ticker.C: //omit
        case instance := <-instanceSyncChan:
            go func() { syncSchema(instance)}()
    }
}
```

Reply:
Done. Made a Go-routine for this.

Reply:
On second thought, this would make send to `instanceSyncChan` blocking when we are in `case <-ticker.C` branch which is synchronous.😵‍💫
Would this be a problem?

Reply:
Yeah, that's the reason I made buffer size 100 initially. It's quite rare to add a new instance. I can make another PR to improve this.